### ISSUE TITLE
switch to ftp-ssl for cddis data

### ISSUE THREAD

**Issue by gregjhogan** [commaai/laika#52]
This will remove the need for generating a `.netrc` file before using this library

see python example for ftp-ssl here:
https://cddis.nasa.gov/Data_and_Derived_Products/CDDIS_Archive_Access.html
```python
from ftplib import FTP_TLS
import sys

email = sys.argv[1]
directory = sys.argv[2]
filename = sys.argv[3]

ftps = FTP_TLS(host = 'gdc.cddis.eosdis.nasa.gov')
ftps.login(user='anonymous', passwd=email)
ftps.prot_p()
ftps.cwd(directory)
ftps.retrbinary("RETR " + filename, open(filename, 'wb').write)
```
should be easy to add since `ftplib` is already used for normal ftp (and the API looks identical)

**Comment by jalotra** [comment]
Was this implemented and merged ?

**Comment by gregjhogan** [comment]
> Was this implemented and merged ?

no, but hopefully it is a pretty straight-forward PR for someone to submit

**Comment by sahandzou** [comment]
Hey!, I fixed the issue in #108 PR

**Comment by sahandzou** [comment]
> > Was this implemented and merged ?
> 
> no, but hopefully it is a pretty straight-forward PR for someone to submit

Hey, i fixed the issue in #109 PR
